MyOwnORM/MyOwnORM/Implementations/CustomDbSetRepository.cs

```diff
@@ -0,0 +1,621 @@
+﻿using MyOwnORM.Helper;
+using MyOwnORM.Interface;
+using MyOwnORM.Reflection;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Reflection.PortableExecutable;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyOwnORM.Implementations
+{
+    public class CustomDbSetRepository<T, TKey> : ICustomDbSetRepository<T, TKey> where T : class
```

Comment:
Now let's make code cleaner. Each CRUD method of repository should only do: call method to get query -> execute with given query, so all methods should not contain string, type definitions and usings

Reply:
Fixed